afkak/client.py

```diff
@@ -826,9 +826,9 @@ def _cancel_timeout(result, dc):
 
         # Make the request to the specified broker
         log.debug('_mrtb: sending request: %d to broker: %r', requestId, broker)
-        d = broker.makeRequest(requestId, request, **kwArgs)
         min_timeout = kwArgs.pop('min_timeout', 0)
         timeout = max(min_timeout, self.timeout)
```

Comment:
Change because min_timeout and timeout need to be set in case the broker.makeRequest() fires immediately?  Or just a style thing?